### PR TITLE
Improve Kiswahili translation

### PR DESCRIPTION
**Pull request by bkmgit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,10 @@ When deploying a server, you'll see the Direct Link as `mumble://murmur-rest:500
 
 *Please note the `docker-compose.yml` file opens ports 50001-50050 for testing. Update as needed.*
 
+## Translations
+
+Translations are welcome. To add or update a translation, please add a file or update a file in [https://github.com/alfg/guildbit/tree/master/app/translations](https://github.com/alfg/guildbit/tree/master/app/translations). For more information, please read the [wiki](https://github.com/alfg/guildbit/wiki/Commands-and-Fixes#updating-translations).
+
```
Comment (alfg) [inline README.md:75]: 👍

File: app/translations/sw/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -0,0 +1,616 @@
+# German translations for PROJECT.
+# Copyright (C) 2014 ORGANIZATION
+# This file is distributed under the same license as the PROJECT project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2014.
+#
+msgid ""
+msgstr ""
```
Comment (alfg) [inline app/translations/sw/LC_MESSAGES/messages.po:7]: This section may need to be updated.

File: README.md (hunk 1 above)
Comment (bkmgit) [reply README.md:75]: Thanks.

File: app/translations/sw/LC_MESSAGES/messages.po (hunk 2 above)
Comment (bkmgit) [reply app/translations/sw/LC_MESSAGES/messages.po]: Thanks. Updated language name and year. Left PROJECT, ORGANIZATION, FIRST AUTHOR <EMAIL@ADDRESS> as is.

**Comment by alfg** [comment]
Thanks @bkmgit! I'll add these soon in the next build.

**Comment by alfg** [comment]
These are now live. Thanks again!

As long as the browser language is set, the translation file should load. You can also override this by setting a `language` cookie to `sw`.

**Comment by bkmgit** [comment]
Thanks.

**Pull request merged**